samples/mqtt5/mqtt5_shared_subscription/README.md

```diff
@@ -4,20 +4,20 @@
 
 This sample uses the
 [Message Broker](https://docs.aws.amazon.com/iot/latest/developerguide/iot-message-broker.html)
-for AWS IoT to send and receive messages through an MQTT connection using MQTT5 using a Shared Subscription.
+for AWS IoT to send and receive messages over a MQTT5 connection using a shared subscription.
 
-MQTT5 introduces additional features and enhancements that improve the development experience with MQTT. You can read more about MQTT5 in the C++ V2 SDK by checking out the [MQTT5 user guide](../../../documents/MQTT5_Userguide.md).
+MQTT5 introduces additional features and enhancements that improve the development experience with MQTT. You can read more about MQTT5 in the Javascript V2 SDK by checking out the [MQTT5 user guide](../../../documents/MQTT5_Userguide.md).
```

Comment:
Should be CPP

Reply:
Ah, good catch! The others probably have this too. I'll update them 👍